To: vim_dev@googlegroups.com
Subject: Patch 7.3.358
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.358 (after 7.3.353)
Problem:    Mouse support doesn't work properly.
Solution:   Add HMT_URXVT. (lilydjwg, James McCoy)
Files:	    src/term.c


*** ../vim-7.3.357/src/term.c	2011-10-26 23:48:17.000000000 +0200
--- src/term.c	2011-11-30 14:42:22.000000000 +0100
***************
*** 1996,2001 ****
--- 1996,2002 ----
  #  define HMT_DEC	4
  #  define HMT_JSBTERM	8
  #  define HMT_PTERM	16
+ #  define HMT_URXVT	32
  static int has_mouse_termcode = 0;
  # endif
  
***************
*** 2031,2036 ****
--- 2032,2042 ----
  	has_mouse_termcode |= HMT_PTERM;
      else
  #   endif
+ #   ifdef FEAT_MOUSE_URXVT
+     if (n == KS_URXVT_MOUSE)
+ 	has_mouse_termcode |= HMT_URXVT;
+     else
+ #   endif
  	has_mouse_termcode |= HMT_NORMAL;
  #  endif
  }
***************
*** 2068,2073 ****
--- 2074,2084 ----
  	has_mouse_termcode &= ~HMT_PTERM;
      else
  #   endif
+ #   ifdef FEAT_MOUSE_URXVT
+     if (n == KS_URXVT_MOUSE)
+ 	has_mouse_termcode &= ~HMT_URXVT;
+     else
+ #   endif
  	has_mouse_termcode &= ~HMT_NORMAL;
  #  endif
  }
*** ../vim-7.3.357/src/version.c	2011-11-30 13:42:40.000000000 +0100
--- src/version.c	2011-11-30 14:45:06.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     358,
  /**/

-- 
I noticed my daughter's Disney-net password on a sticky note:
"MickeyMinnieGoofyPluto".  I asked her why it was so long.
"Because they say it has to have at least four characters."

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
